packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -124,6 +138,41 @@ export async function uploadArtifactToS3(
     }
 }
 
+export async function resumeTransformationJob(jobId: string, userActionStatus: TransformationUserActionStatus) {
+    try {
+        const apiStartTime = Date.now()
+        const response = await codeWhisperer.codeWhispererClient.codeModernizerResumeTransformation({
+            transformationJobId: jobId,
+            userActionStatus, // can be "COMPLETED" or "REJECTED"
+        })
+        if (response) {
+            telemetry.codeTransform_logApiLatency.emit({
+                codeTransformApiNames: 'ResumeTransformation',
+                codeTransformSessionId: CodeTransformTelemetryState.instance.getSessionId(),
+                codeTransformJobId: jobId,
+                codeTransformRunTimeLatency: calculateTotalLatency(apiStartTime),
+                codeTransformRequestId: response.$response.requestId,
+                result: MetadataResult.Pass,
+            })
+            // always store request ID, but it will only show up in a notification if an error occurs
+            return response.transformationStatus
+        }
+    } catch (e: any) {
+        const errorMessage = (e as Error).message
+        getLogger().error(`CodeTransformation: ResumeTransformation error = ${errorMessage}`)
+        telemetry.codeTransform_logApiError.emit({
```

Comment:
why is this a separate metric? the `codeTransform_logApiLatency` metric could have a result=fail with `reason` field set. there is no need to have separate "failure" metrics for each event.

Reply:
This is a standing pattern. Closing out.